golem_messages/message.py

```diff
@@ -96,7 +96,7 @@ def deserialize_cannot_compute_task(key, value):
 class Message():
     """ Communication message that is sent in all networks """
 
-    __slots__ = ['timestamp', 'encrypted', 'sig', '_payload', '_raw']
+    __slots__ = ['timestamp', 'encrypted', 'sig', '_raw']

```

Comment:
`payload` is still passed to `__init__` (line 274)

Reply:
Good point. Created #54